src/main/java/ru/my/Mocito/employeeController/EmployeeController.java

```diff
@@ -0,0 +1,35 @@
+package ru.my.Mocito.employeeController;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import ru.my.Mocito.Employee;
+import ru.my.Mocito.employeeService.EmployeeImpl;
+
+
+@RestController
+@RequestMapping("/employee")
+public class EmployeeController {
+    private final EmployeeImpl employeeImpl;
```

Comment:
```suggestion
    private final Iemployee employeeImpl;
```
инъекцию надо делать по интерфейсу, а не реализации